listeners/assistant/assistant.py

```diff
@@ -118,23 +118,23 @@ def respond_in_assistant_thread(
 
         returned_message = call_llm(messages_in_thread)
 
-        stream_response = client.chat_startStream(
+        streamer = client.chat_stream(
             channel=channel_id,
             recipient_team_id=team_id,
             recipient_user_id=user_id,
             thread_ts=thread_ts,
         )
-        stream_ts = stream_response["ts"]
 
-        # use of this for loop is specific to openai response method
+        # Loop over OpenAI response stream
+        # https://platform.openai.com/docs/api-reference/responses/create
```

Comment:
I added commit 15d6bce to keep the comments consistent between the `assistant` and `app_mention` files.